src/Controller/TaskController.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Task;
+use App\Form\TaskType;
+use App\Repository\TaskRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
+
+class TaskController extends AbstractController
```

Comment:
Ici tu peux renseigner via annotation ce qui préfix la route et le nom des méthodes afin de ne pas avoir à le répéter sur chaque méthode :

```suggestion
#[Route('/task', name: 'app_task_')]
class TaskController extends AbstractController
```

Ce qui par exemple te permet après sur ta méthode edit de faire :

```php 
#[Route('/edit/{id}', name: 'edit', methods: ["GET", "POST"])]
```